dashboards/scylladb/metadata.yaml

```
@@ -8,3 +8,13 @@ sample_dashboards:
     related_integrations:
       - id: scylladb
         platform: GKE
+        version: "1"
+    category: ScyllaDB
+    id: scylladb-prometheus
```

```suggestion
    id: scylladb-prometheus-v2
```

Similarly the id for the first dashboard should be `scylladb-prometheus-v1` from the corresponding json file's name.